elements/chameleon-common/post-install.d/99-lockdown-firewall

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# Delete existing rules
+ufw --force enable
+for i in {4..1}; do ufw --force delete $i; done
```

Comment:
Why is this counting backwards? just wondering

Reply:
Once I delete rule 1, there are only 3 rules left, so no more rule 4. I either have to count backwards, or delete rule 1 4 times. If I delete rule 1 then 2 then 3... There is no more 3 by the time I get to it.  :)